src/components/UnlockWallet/index.tsx

```diff
@@ -30,6 +31,49 @@ function warnUser(title: string, message?: string | undefined): void {
     message,
   });
 }
+function changeId(chainName : any) {
+  var chains =  JSON.parse(JSON.stringify(internalAddressesPerNetwork))
+  const chain = chains[chainName]
+  try {
```

Comment:
Replace mega-`try`-blocks with a `try`'d call to another function.

Reply:
Since the `catch` will never be reachable, I removed the exception